docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -138,6 +139,7 @@ following works:
 - github.com/karrick/godirwalk [BSD 2-Clause "Simplified" License](https://github.com/karrick/godirwalk/blob/master/LICENSE)
 - github.com/kballard/go-shellquote [MIT License](https://github.com/kballard/go-shellquote/blob/master/LICENSE)
 - github.com/klauspost/compress [BSD 3-Clause Clear License](https://github.com/klauspost/compress/blob/master/LICENSE)
+- github.com/kylelemons/godebug [Apache License](https://github.com/kylelemons/godebug/blob/master/LICENSE)
```

Comment:
Looks like we can remove this as it's not part of the project.

Reply:
But we use it as an eventual dependency, if I run the command `go mod why github.com/kylelemons/godebug/pretty` I will have the result:
```
# github.com/kylelemons/godebug/pretty
github.com/influxdata/telegraf/plugins/outputs/azure_data_explorer
github.com/Azure/azure-kusto-go/kusto/ingest
github.com/kylelemons/godebug/pretty
```

And if I run the command `make check-deps` when I remove the line 142, I get the result:
```
./scripts/check-deps.sh
--- /var/folders/km/r01j6n215ld0_ps3k_x9h08w0000gn/T/tmp.4MvOyb57/LICENSE_OF_DEPENDENCIES.md    2021-07-07 12:59:28.000000000 +1000
+++ /var/folders/km/r01j6n215ld0_ps3k_x9h08w0000gn/T/tmp.4MvOyb57/HEAD  2021-07-07 12:59:28.000000000 +1000
@@ -139,0 +140 @@
+github.com/kylelemons/godebug


The docs/LICENSE_OF_DEPENDENCIES.md file does not contain the expected entries.

Lines prefixed with '+' should be added to LICENSE_OF_DEPENDENCIES.md and '-'
lines should be removed.

Include a link to the appropriate licenses for any additions.
make: *** [check-deps] Error 1
```